src/core/models/interfaces/analytic.ts

```diff
@@ -0,0 +1,50 @@
+export interface Analytic {
+  series: {
+    period: 'total' | 'annual' | 'semiAnnual' | 'quarterly' | 'monthly' | 'weekly' | 'daily' | 'hourly';
+    start: string;
+    end: string;
+    rows: {
+      dimensions: {
+        name: 'budget';
+        path: string;
+      }[];
+      metric: 'Actuals' | 'Budget' | 'Forecast' | 'PaymentsOnChain' | 'PaymentsOffChainIncluded';
```

Comment:
The metrics should be types/enums so they can be reused

Reply:
Done! check: https://github.com/makerdao-ses/ecosystem-dashboard/pull/1117/commits/dcb1808c34b8389cc547be03221a06f17d27209d